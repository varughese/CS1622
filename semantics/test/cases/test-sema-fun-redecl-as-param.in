void foo(void)
{
}

void bar(int foo)
{
}

void main(void)
{
}
