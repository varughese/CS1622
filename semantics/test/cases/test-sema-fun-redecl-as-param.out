[program
  [fun-declaration
    [void]
    [foo]
    [params]
    [compound-stmt 
    ]
  ]
  [fun-declaration
    [void]
    [bar]
    [params 
      [param [int] [foo]]]
    [compound-stmt 
    ]
  ]
  [fun-declaration
    [void]
    [main]
    [params]
    [compound-stmt 
    ]
  ]
]
